### ISSUE TITLE
Cuba Rio Damuji-class OPV x2

### ISSUE THREAD

**Issue by cdnice55** [PygmalionOfCyprus/cmo-db-requests#770]
### DB Selector

DB3K

### Hypothetical

No

### Name

BP 390 Rio Damuji

### Class

Rio Damuji-class OPV

### Country

Cuba

### Service

Navy

### In Commission

2007-

### Length (m)

106.86

### Beam (m)

14.61

### Draft (m)

5.63

### Standard Displacement (t)

3200

### Complement

40

### Primary Propulsion Type

Diesel

### Propulsion

Generic Diesel generators

### Flank Speed (kt)

15

### Range (nm) at Speed (kt)

Not available

### Sensors

- 1x Navigation Radar

### Mounts

- 3 x 2 - 25/82 2M-3M
- 2 x 1 P-15M SSM (2 P-15M)
- 1 x 2 - 57/76 ZU-57-2

### Aircraft & Docking Facilities

- 1x Pad for medium helicopter (Mil Mi-14)

### Comms/Datalinks

Standard for Cuban Navy

### Ship Codes

Built to Mercantile Standards

### Sources

http://www.navypedia.org/ships/cuba/cub_es_la_habana.htm
https://en.wikipedia.org/wiki/Rio_Damuji-class_frigate
http://www.globalsecurity.org/military/world/cuba/rio-damuji.htm

![300px-Rio_Damuji_n°_390](https://user-images.githubusercontent.com/99632305/159827881-18faa22a-8ab1-4b36-80f6-9b51d89e842e.jpg)

**Comment by PygmalionOfCyprus** [comment]
Fill #460

**Comment by cdnice55** [comment]
Yeah, I think they literally add whatever they can get their hands on to these ships.

**Comment by PygmalionOfCyprus** [comment]
Some guy on Reddit called it a "sea technical" and I think that's a very apt description.

I do wonder how many of the weapons are actually seriously intended for use and how many are just for show. Like, okay, great, you've got ancient P-15Ms, but no long-range radar. Likewise, AK-630s but no targeting radar (you can use manual guidance but good luck actually shooting anything down that way). Decoys with no ESM? RBU-1200s with no sonar?

Feels like the sort of thing you use to show off domestically and to conduct low-intensity EEZ patrols where the biggest threat is some old junker fishing illegally rather than a true "domestic surface combatant."

**Comment by tbrandon517** [comment]
Found an apparent image of the RBU's

https://preview.redd.it/vs9gobs7s7o81.png?auto=webp&s=57a1375e38294fb3310910985772b3d79ca6c8a7

**Comment by PygmalionOfCyprus** [comment]
Added the RBUs. I wonder if they're just using them as a sort of naval MLRS, rather than for ASW.

**Comment by tbrandon517** [comment]
> Added the RBUs. I wonder if they're just using them as a sort of naval MLRS, rather than for ASW.

I think Oryx blog did a piece on ground equipment where they were using trailer mounted RBU's as Coast Defense Artillery so it wouldn't surprise me. Or they could blast narco-subs with them.

**Comment by PygmalionOfCyprus** [comment]
"Narco-subs" are almost always just semi-submersibles, so it'd be the same mechanism to blow one up as any other surface ship...provided you can hit it with your 120 yard RBU-1200 spread. Or even find it. Which, considering even the USCG has trouble doing, I don't have high hopes for the "strap-a-torpedo-tube-to-a-pontoon-float" Navy...

**Comment by cdnice55** [comment]
Can't imagine the deck house and decking was built for this:
![f0119041](https://user-images.githubusercontent.com/99632305/169614878-096e99a6-95a7-4c39-89f1-5b6633469295.jpg)
![Cubapatrulhalançandomíssil](https://user-images.githubusercontent.com/99632305/169614746-8bd6385d-de30-4d0e-b738-3ef3787ddf11.png

Sideview of RBU's firing I did not see them in other photo's showing the SSM's but this shows them firing with the SSM's there as well:
![C1WvnjWWQAAWHso](https://user-images.githubusercontent.com/99632305/169615071-78238ddd-0bae-438c-a8dc-918842300140.jpg)

Found these 6 pics that span 2014 to 2021 as well.
![RIO_DAMUJI](https://user-images.githubusercontent.com/99632305/169616027-01bfc5b6-710c-4a54-9342-e7b47443c2e1.jpg)

Speed boat attached on the rear here:
![B_v1EZEWYAEQUsu](https://user-images.githubusercontent.com/99632305/169616123-e12d4ba2-c7f8-44be-972c-6738db130ac3.jpg)

![5fZfehA](https://user-images.githubusercontent.com/99632305/169616621-bf8244bb-c95b-466d-8d43-50b6656a81c9.jpg)
![imK1roZ](https://user-images.githubusercontent.com/99632305/169616787-5c5ebaa7-bdc3-4584-acbc-5b42be41a353.jpg)
![EZZ5n6S](https://user-images.githubusercontent.com/99632305/169616806-a8ab4633-aa9d-43ed-89dc-c1f3e717b1f5.jpg)

More boats aboard:
![f0089807](https://user-images.githubusercontent.com/99632305/169616717-2de19367-c554-4ca7-95e9-ae0072c122e9.jpg)

This ship makes my brain hurt....

**Comment by PygmalionOfCyprus** [comment]
Stuff like this is why I love third world militaries. It's like Mad Max out there.

**Comment by tbrandon517** [comment]
@PygmalionOfCyprus still haven't seen a photo of the T-55 turret but it does seem at least one has an AK-725 installed.

https://beyondthesprues.com/Forum/index.php?topic=5289.0
https://weaponsystems.net/system/539-57mm+AK-725
